portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/retry/RetryingCallable.java

```diff
@@ -71,9 +71,17 @@ public T call() throws RetryException {
         // TODO: do we want to reset anything (eg, number of retries) if we see a different
         // RetryStrategy?
         RetryStrategy strategy = retryStrategyLibrary.checkoutRetryStrategy(e);
+        monitor.info(
```

Comment:
This feels more like a debug log to me, but I don't feel strongly about it if you want to leave it as info.

Reply:
That makes sense, we already have the severe on line 69